stream/src/js/stream.js

```diff
@@ -1,10 +1,59 @@
 $(document).ready(function() {
+    actionkit.forms.contextRoot = 'https://go.peoplepower.org/context/';
+    actionkit.forms.initPage();
+    actionkit.forms.initForm('act');
+
+    // I'll admit. Not the cleanest way but it'll work!
+    $('#signup-form').submit(function(e) {
+        var form = $(e.target);
+
+        var handler = function(form, success) {
```

Comment:
Success and failure paths share no code, this could be two functions and be a bit cleaner